eventLab.js

```diff
@@ -0,0 +1,23 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let form = document.querySelector("form");
+  form.addEventListener("submit", e => {
+    e.preventDefault();
+    let li = document.createElement("li");
+    let task = document.querySelector("#task");
+    let ul = document.querySelector("#list");
+    let newTask = task.value;
+    let p = document.createElement("p");
+    if (newTask === "") {
```

Comment:
This works but you can also try `newTask.trim()` this removes empty spaces, then it returns a value. So if the user has empty spaces its going to return `""` .